examples/layers/JSONLayers/Region.json

```diff
@@ -7,7 +7,8 @@
 	"style"      : "",
 	"projection" : "EPSG:4326",
 	"transparent" : true,
-	"bbox"      : [-61.80983300000002, -21.38963079935857, 55.83665389999998, 51.0890011990783],
+    "unit"      : 1,
```

Comment:
Does itowns codebase support projection|crs anywhere? (hint: the projection defined in Region.json is *wrong*, according to the `GetCapabilities` it should be `<CRS>EPSG:3857</CRS>`).

But OK, I'll remove `unit` and add a list of know crs and deduce the unit from the projection.